ga4gh/client.py

```diff
@@ -466,7 +466,7 @@ def searchExpressionLevel(self, expressionLevelId=None,
             protocol.SearchExpressionLevelResponse)
 
     def searchFeatureGroup(self, rnaQuantificationId=None,
-                           featureGroupId=None, threshold=0.0):
+                           featureGroupId=None):
```

Comment:
since sequence annotation features can exist and be searched in other contexts than rna quantification, should the rnaQuantificationId still be in this method?